easybuild/easyconfigs/p/Python/Python-3.6.4-iomkl-2018.02.eb

```diff
@@ -0,0 +1,221 @@
+name = 'Python'
+version = '3.6.4'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'iomkl', 'version': '2018.02'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['7dc453e1a93c083388eb1a23a256862407f8234a96dc4fae0fc7682020227486']
+patches = ['Python-3.6.3-icc_segfault_workaround.patch']
```

Comment:
@schiotz Please also include a SHA256 checksum for the patch file (`--inject-checksums` will do this for you)